payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -134,9 +129,37 @@ internal class PaymentSheetViewModel @Inject internal constructor(
 
     private val stripeIntentValidator = StripeIntentValidator()
 
+    @VisibleForTesting
+    internal var googlePayPaymentMethodLauncher: GooglePayPaymentMethodLauncher? = null
+
+    @VisibleForTesting
+    internal val googlePayLauncherConfig: GooglePayPaymentMethodLauncher.Config? =
+        args.googlePayConfig?.let { config ->
+            if (config.currencyCode == null && !isProcessingPaymentIntent) {
+                logger.warning(
+                    "GooglePayConfiguration.currencyCode is required in order to use " +
+                        "Google Pay when processing a Setup Intent"
+                )
+                null
+            } else {
+                GooglePayPaymentMethodLauncher.Config(
```

Comment:
Do you have plans to update the `StripeGooglePayContract` to the `GooglePayPaymentMethodLauncherContract`?

Reply:
No, but I will clean up the GooglePay* classes once I've migrated the custom flow too.